config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -59,14 +57,18 @@ presubmits:
         resources:
           requests:
             cpu: 2
-            memory: "9Gi"
+            memory: "1Gi"
+          limits:
+            cpu: 2
+            memory: "1Gi"
```

Comment:
FWIW: we're usually CPU constrained scheduling wise, and you may see some issues with OOM at only 1Gi compiling (YMMV), I would recommend something more like 1Gi /core for the floor (less won't really hurt the infrastructure, but may cause unnecessary issues with your job).

This is something we need to discuss more in sig-k8s-infra and better document (machine shapes and actual schedulable resources)

Reply:
Thanks! I was lowering it here because we inherited the original value from the CAPZ jobs which use kind. Windows no longer uses kind so thought could help reduce resource usage. I will use the 1gig/core for now.

This was a topic we were discussing in CAPZ. Is there or will there be the ability to see the actual usage of a given job so we could right size over time?

Reply:
>  Is there or will there be the ability to see the actual usage of a given job so we could right size over time?

Yes! The community clusters have https://monitoring-eks.prow.k8s.io/ and https://monitoring-gke.prow.k8s.io/ (WIP)

Reply:
(these map to `cluster: eks-prow-build-cluster` and `clsuter: k8s-infra-prow-build`)